src/Gui/CommandView.cpp

```diff
@@ -1969,6 +1971,51 @@ bool StdViewScreenShot::isActive()
     return isViewOfType(Gui::View3DInventor::getClassTypeId());
 }
 
+//===========================================================================
+// Std_ViewLoadImage
+//===========================================================================
+DEF_STD_CMD(StdViewLoadImage)
+
+StdViewLoadImage::StdViewLoadImage()
+  : Command("Std_ViewLoadImage")
+{
+    sGroup      = "Standard-View";
+    sMenuText   = QT_TR_NOOP("Load picture...");

```

Comment:
Any reason to use "picture" instead of "image"? It feels a little informal.

Reply:
Actually _image_ was the right term but the screen shot command above used the term _picture_ for consistency reasons. We can change then both commands to use the term _image_.